src/main/java/core/basesyntax/operation/ReturnOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class `ReturnOperation` should be placed in the `strategy` package, as per the checklist requirement to group classes with common logic/purpose.